src/test/kotlin/no/fdk/concept_catalog/service/SkosApNo.kt

```diff
@@ -153,8 +121,8 @@ class SkosApNo {
             .thenReturn(BEGREP_6.copy(definisjon = BEGREP_6.definisjon?.copy(kildebeskrivelse = Kildebeskrivelse(forholdTilKilde = ForholdTilKildeEnum.SITATFRAKILDE, kilde = listOf(URITekst(uri = "https://an invalid uri", tekst = "Testdirektoratet"))))))
         val modelInvalidURI = assertDoesNotThrow { skosApNo.buildModelForConcept(BEGREP_6.ansvarligVirksomhet.id, BEGREP_6.id!!) }
 
-        val sourceNullURI = modelInvalidURI.listObjectsOfProperty(SKOSNO.definisjon).toList().first().asResource().getPropertyResourceValue(DCTerms.source)
+        val sourceNullURI = modelInvalidURI.listObjectsOfProperty(EUVOC.xlDefinition).toList().first().asResource()
         assertFalse { sourceNullURI.hasProperty(RDFS.seeAlso) }
-        assertTrue { sourceNullURI.hasProperty(RDFS.label) }
+        assertTrue { sourceNullURI.hasProperty(DCTerms.source) }
```

Comment:
Det hadde vært bedre om du fikk sjekket at det er er verdien fra tekstfeltet som har blitt lagt til, siden begge ender opp som dct:source
```suggestion
        assertTrue { sourceNullURI.hasProperty(DCTerms.source) }
```

Reply:
men husker ikke helt hvordan det blir gjort, skal sjekke litt